### ISSUE TITLE
Create View Individual Listing Component

### ISSUE THREAD

**Issue by CaelumD25** [UVicMartletplace/martletplace#51]
## Feature Description

- Params:
  - JSON Object, Listing Object(OpenAPI spec)

relates: #xxx
blocked by: #xxx

## Problem You're Trying to Solve

When a user clicks on a particular listing, this is the view they will see. Note that the reviews component will be a separate ticket. 

## Related External Resources

![Image](https://github.com/UVicMartletplace/martletplace/assets/72957712/77e166b2-9538-4fbe-aa45-4981c9a6a39e)
